gatsby-node.js

```diff
@@ -1,8 +1,10 @@
 const path = require(`path`)
 const mangoSlugfy = require(`@mangocorporation/mango-slugfy`)
 const { createRemoteFileNode } = require(`gatsby-source-filesystem`)
+const { curry, __, pipe, map, isEmpty, prop, flatten } = require('ramda');
 
-const paginationPath = (page, totalPages, prefix = `/`) => {
+const paginationPath = (page, totalPages, pfx) => {
```

Comment:
All the parameters are not abbreviated but the last, would be more consistent to keep the full name "prefix"